src/js/vic-v2/components/PhotoField.jsx

```diff
@@ -365,74 +396,93 @@ export default class PhotoField extends React.Component {
-      const containerWidth = this.refs.cropper.getContainerData().width;
-      // const containerWidth = this.refs.container.offsetWidth;
-      const smallScreen = isSmallScreen(this.state.windowWidth) || onReviewPage(this.props.formContext.pageTitle);
-      const left = smallScreen ? (containerWidth / 2) - 120 : (containerWidth / 2) - 150;
-      this.refs.cropper.setCropBoxData({
-        top: 0,
-        left,
-        height: smallScreen ? SMALL_CROP_BOX_SIZE : LARGE_CROP_BOX_SIZE,
-        width: smallScreen ? SMALL_CROP_BOX_SIZE : LARGE_CROP_BOX_SIZE
-      });
+      // sometimes this callback is called before the new cropper has mounted
+      // check if the cropper instance has canvasData before setting its values
+      if (!_.isEmpty(this.refs.cropper.getCanvasData())) {
+        // center the cropbox using container width
+        const containerWidth = this.refs.cropper.getContainerData().width;
+        const smallScreen = isSmallScreen(this.state.windowWidth) || onReviewPage(this.props.formContext.pageTitle);
+        const left = smallScreen ? (containerWidth / 2) - 120 : (containerWidth / 2) - 150;
+        this.refs.cropper.setCropBoxData({
+          top: 0,
+          left,
+          height: smallScreen ? SMALL_CROP_BOX_SIZE : LARGE_CROP_BOX_SIZE,
+          width: smallScreen ? SMALL_CROP_BOX_SIZE : LARGE_CROP_BOX_SIZE
+        });
 
-      // use the cropbox dimensions to force canvas into default position
-      const { height: cropBoxHeight, width: cropBoxWidth, left: cropBoxLeft } = this.refs.cropper.getCropBoxData();
-      const { width: oldCanvasWidth, height: oldCanvasHeight, naturalHeight, naturalWidth } = this.refs.cropper.getCanvasData();
-      // tall images take the full width of the cropBox
-      if (naturalHeight > naturalWidth) {
-        this.refs.cropper.setCanvasData({
-          width: cropBoxWidth,
-          left: cropBoxLeft,
-          top: 0
+        const defaultCanvasData = getCanvasDataForDefaultPosition({
+          canvasData: this.refs.cropper.getCanvasData(),
+          cropBoxData: this.refs.cropper.getCropBoxData(),
+          containerWidth
         });
-        // wide images take the full height of the cropBox
-        // to center, uses the ratio of cropBoxHeight / cavasHeight
-      } else {
-        this.refs.cropper.setCanvasData({
-          height: cropBoxHeight,
-          top: 0,
-          left: (containerWidth - (cropBoxHeight / oldCanvasHeight * oldCanvasWidth)) / 2
+
+        this.refs.cropper.setCanvasData(defaultCanvasData);
+
+        this.setZoomToDefaultRatio();
+
+        // update warning messages
+        const boundedCanvasData = getBoundedCanvasPositionData({
+          canvasData: this.refs.cropper.getCanvasData(),
+          cropBoxData: this.refs.cropper.getCropBoxData(),
         });
+
+        this.updateBoundaryWarningAndButtonStates(boundedCanvasData);
       }
+    });
+  }
 
+  setZoomToDefaultRatio = () => {
+    window.requestAnimationFrame(() => {
```

Comment:
Ensures that any rotations / movements are finished before determining the min zoom ratio.